src/assets/productData.js

```diff
@@ -0,0 +1,37 @@
+import faker from "faker";
```

Comment:
Instead of keeping data inside of assets folder. You can give it a different name say db or data.
assets  basically includes images or svgs to be used.